assets/scripts/passwd-strength-indicator.js

```diff
@@ -1,10 +1,12 @@
 (function (window, document) {
   const indicator = document.getElementById('password-strength')
+  const submit = document.getElementById('login-submit')
 
   document.getElementById('password').addEventListener('input', function(event) {
     const strength = getStrength(event.target.value)
     indicator.value = parseInt(strength.percentage, 10)
     indicator.setAttribute('class', `pw-${strength.label}`)
+    submit[strength.label == 'weak' ? 'setAttribute' : 'removeAttribute']('disabled', '')
```

Comment:
Actually, I think we should keep the password helper as an external module, and keep it at the exact same format of the one in [cozy-settings](https://github.com/cozy/cozy-settings/blob/master/src/lib/passwordHelper.js). It could be used in `passwd-strength-indicator` *and* in `async-submit`, to get the password strength. This strength could be use in `async-submit` to validate the password and then enable/disable the submit button *and* disallow form submission if the password is not valid. Do you think that we could mix those two logics without too much explicit if/else ? Actually I am a little bit surprised that both login form and password reset form are using the exact same script.